mw-webapp/src/dataAccessLogic/WayPreviewDAL.ts

```diff
@@ -79,8 +79,19 @@ export class WayPreviewDAL {
   /**
    * Get User own ways preview
    */
-  public static async getOwnWaysPreview(uuid: string): Promise<WayPreview[]> {
-    const waysDTO = await WayService.getOwnWaysDTO(uuid);
+  public static async getUserWaysPreview(uuid: string, type: "OwnWays" | "MentoringWays"): Promise<WayPreview[]> {
+
```

Comment:
Add to comment that you get not only own ways but also mentoring ways

Reply:
Done